sys-devel/clang/Manifest

```diff
@@ -1,4 +1,5 @@
 DIST cfe-4.0.0.src.tar.xz 10900916 SHA256 cea5f88ebddb30e296ca89130c83b9d46c2d833685e2912303c828054c4dc98a SHA512 a0d9972ec337a5c105fcbe7abc4076ba1e580f28908a3318f43bbfe59143f446ed5b78dad210f624145d7e5a3d56c15bfead78826c068422b60120fa1cfa482a WHIRLPOOL fe04b6955b82915bba09726947fceff92e67ffaac97de4b8c32c18546262f60a4307fdaccd3c9540710392658ed47f3bcfe44791de8d7d30786d56576f339aee
+DIST clang-4.0.0-flang.patch.xz 12596 SHA256 af5529c1405eb1679d083f8cb7ad232ff0727ec03fb51775e769c652a721fd82 SHA512 e60718ae6d232e9033c8d346c07688c67b6bb0ef2b1893da69d4359d282a3ab2f1737a3177b85899557b04d3fad6310c319220873362f6d49d1edf9393583523 WHIRLPOOL 19f3dac0ef84b89a2c5eb2f93276eeb14a34d8b175bc8e0b137c531b9f40d21fc2bf1ac700f4f574dd578f4d2b86426ebf53fde957c4348daedde165d8a8160c
```

Comment:
The patch isn't big: https://github.com/junghans/clang/compare/release_40...junghans:flang_release_40
and it mainly add a fortran case everywhere.

Half of it has been merge between 3.9 and 4.0, the rest hopefully will go into the next release.